src/DataDictionary/NodeCategories/helper.js

```diff
@@ -1,3 +1,5 @@
+import * as d3 from 'd3-scale';
```

Comment:
can we just import `schemeCategory20` rather than the whole pack since we only need that